src/onesignal.ts

```diff
@@ -81,9 +88,35 @@ export async function sendReadingReminderPush(
     'push',
     'reminder',
   );
-  push.chrome_web_badge =
-    'https://daily-now-res.cloudinary.com/image/upload/v1672745846/public/dailydev.png';
-  push.chrome_web_icon =
-    'https://daily-now-res.cloudinary.com/image/upload/s--9vc188bS--/f_auto/v1712221649/1_smcxpz';
+  push.chrome_web_badge = chromeWebBadge;
+  push.chrome_web_icon = chromeWebIcon;
+  await client.createNotification(push);
+}
+
+export async function sendStreakReminderPush(
+  userIds: string[],
+  at: Date,
+): Promise<null | OneSignal.CreateNotificationSuccessResponse> {
+  if (!appId || !apiKey) return null;
+  const push = new OneSignal.Notification();
+  push.app_id = appId;
+  push.include_external_user_ids = userIds;
+  push.send_after = at.toISOString();
```

Comment:
Can't we just omit this one actually to send it directly?

Reply:
Take a look now, I removed it